gcs-action/package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "gcs-action",
```

Comment:
Same note as above. Since this action doesn't encompass every possible thing GCS can do (and it shouldn't), I think we need to name it more appropriately like `cloud-storage-uploader`.

Reply:
Done